raiden/api/rest.py

```diff
@@ -154,6 +156,7 @@
         ChannelBlockchainEventsResource,
     ),
     ("/_debug/raiden_events", RaidenInternalEventsResource),
+    ("/_testnet/tokens/<hexaddress:token_address>/mint", MintTokenResource, "tokensmintresource"),
```

Comment:
Why did we go for `_testnet` rather than adding the endpoint under the `_debug` namespace?

Reply:
Kelsos and Lefteris suggested in the discussion to have a seperate name space for testing APIs. `_debug` wouldn't make much sense since this is not exactly for debugging.